conda_lock/src_parser/__init__.py

```diff
@@ -287,7 +278,7 @@ def seperator_munge_get(
 
     for dep, root in root_requests.items():
         source = requested[root]
-        target = seperator_munge_get(planned, dep)
+        target = seperator_munge_get(planned, normalize_conda_name(dep).lower())
```

Comment:
I think this place is too common to normalize the name of a dep here. We may need to update the dep model to have multiple names in it for pypi names and conda names

Reply:
Makes sense. Maybe we should close this one and start over? I have to move to another project but I can try to tackle that when I get more free time.

Reply:
Nah, i'll take this one over.  The core idea is right, just not in the correct spot.